src/http/test/http_test.cpp

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the Apache 2.0 License.
 #define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
```

Comment:
what made these two tests suddenly need the `DOCTEST_` prefix?

Reply:
`logger::config::level() = logger::INFO;` in some of the test cases to make them less verbose.